Adds operator token requirement to org and user docs in v2 (#5386)
influxdata/docs-v2: content/influxdb/v2/admin/users/create-user.md

To create a new user, you must have an [operator token](/influxdb/v2/reference/glossary/#token).
For more information, see how to [create an operator token](/influxdb/v2/admin/tokens/create-token/#create-an-operator-token).
Use the [`influx user create` command](/influxdb/v2/reference/cli/influx/user/create)
and provide the following:

- An [operator token](/influxdb/v2/admin/tokens/#operator-token) using your
  [`influx` CLI connection configuration](/influxdb/v2/reference/cli/influx/#provide-required-authentication-credentials),
  `INFLUX_TOKEN` environment variable, or the `--token, -t` flag.
- The [organization name or ID](/influxdb/v2/admin/organizations/view-orgs/) to
  add the new user to.
- A username for the new user with the `--name, -n` flag.

> - _Optional:_ A password for the new user. If no password is provided, the new
>   user will be prompted to provide one.

Reviewer: ```suggestion
- _Optional_: the `--password, -p` flag and a password for the user. If you don't provide a password, the new user will be prompted to provide one.
```
